spec/factories/user.rb

```diff
@@ -5,6 +5,7 @@
     school { School.find_by(name: 'test') || create(:school, :current) }
     title { Faker::Lorem.words(number: 3).join(' ') }
     time_zone { ENV['SPEC_USER_TIME_ZONE'] || 'Asia/Kolkata' }
+    external_id { 'c2896708-78c1-40f1-9f17-99f132335b47' }
```

Comment:
I would generate it unique for each User record instead of hard coding it. But ignore for this project now.

Reply:
Indeed, that would be better.